README.md

```diff
@@ -32,6 +32,7 @@ The theme is currently in alpha, and does not have released versions. To install
 
 ## Additional features
 
+- Custom Score support for intergration with Simply Training, from the Simply Love beta branch

```

Comment:
Nit: I'd not mention the beta branch, as it'll graduate eventually

Reply:
Eh, if we don't mention the beta branch it sounds like it's not in SL at all, like the search in density graph, and I didn't feel that was correct attribution-wise. I see your point about the readme becoming wrong as soon as SL releases, though.

I guess we could just leave it out, but then it sounds like it's not a feature we have at all since it's not in either SL 4.9 or the "Additional features" thing. Which makes me wonder how we'll handle it _generally_ now that we've decided to cherry-pick from SL, actually. 😅 

I suggest it's a larger discussion we'll be having soon™.